docs/root/intro/arch_overview/operations/hot_restart.rst

```diff
@@ -5,8 +5,8 @@ Hot restart
 
 Ease of operation is one of the primary goals of Envoy. In addition to robust statistics and a local
 administration interface, Envoy has the ability to “hot” or “live” restart itself. This means that
-Envoy can fully reload itself (both code and configuration) without dropping any connections. The
-hot restart functionality has the following general architecture:
+Envoy can fully reload itself (both code and configuration) without *immediately* dropping connections.
```

Comment:
im not sure if this really resolves the problem you are pointing out, more it suggests that the connections *might* be dropped at an indeterminate time in the future

would changing to:

"... without dropping existing connections..." 

capture the functionality ?

Reply:
I would suggest something like "without dropping existing connections during the drain process" and then linking "drain process" to draining documentation either below or elsewhere?

/wait

Reply:
Thanks 

@mattklein123 that's a good way of phrasing it.

Our use case for Envoy is for proxing long standing TCP connections so some implication that existing connections will be eventually dropped is useful.

Reply:
Apologies for the delay, I got distracted then forgot to make the markups. I've now made the change to your sugesttion above